fastlane/Fastfile

```diff
@@ -31,8 +31,8 @@ end
 
 desc "Start a new release"
 lane :release do |options|
-  # Ensure we are on the main branch
-  ensure_git_branch(branch: 'main')
+  # Ensure we are on `develop` or `main`
+  ensure_git_branch(branch: '^main$|^develop$')
```

Comment:
I'm not sure this would work for hotfixes tho 🤔 Usually when we create a hotfix it is going to be from a tag, and not from the "main" branch. So I'm not sure if this "ensure_git_branch" is even worth being here 🤔 Actually probably an "ensure_is_not_main" is actually more useful since I don't see a case where we want to release from main. And this will avoid we make a mistake because of muscle memory 😄 (Since we have been releasing from main until now)

Reply:
Oh, you are right... Let me remove this, because it might not be a good check to have

Reply:
@polqf I actually spotted one more thing 🍎 https://github.com/GetStream/stream-chat-swift/blob/53d453f732db50e9b7598e70a32c82d8a9852674/fastlane/Fastfile#L101

Do we still need that "base: main"? 🤔

Reply:
Yes, because releases are merged to main

Reply:
Ahhh yes! Sorry totally misread that. Though the PR was being created from "main" xD